security/s2a/internal/crypter/aesgcm.go

```diff
@@ -39,33 +39,33 @@ type aesgcm struct {
 	keySize int
 }
 
-// NewAESGCM creates an AES-GCM crypter instance. Note that the key must be
+// newAESGCM creates an AES-GCM crypter instance. Note that the key must be
 // either 128 bits or 256 bits.
-func NewAESGCM(key []byte) (S2AAeadCrypter, error) {
+func newAESGCM(key []byte) (s2aAeadCrypter, error) {
```

Comment:
Will the half connection implementation be in `crypter` package?

Reply:
Yes